### ISSUE TITLE
Example of how to contribute to Angle documentation - Title

### ISSUE THREAD

**Issue by tutacrypto** [AngleProtocol/angle-docs#39]
# Example of how to contribute to Angle documentation

## About this guide
This guide should help non-devs people get their translations included faster in the documentation. 

If you're familiar with Github, you can simply clone the repository, and create a pull request with the translated files. Please make sure you change the SUMMARY.md document with the link to your translation. 

## Translating 
The first thing to do when translating or updating some documentation, is to write your text and have it reviewed by at least one native speaker from the community who has a good understanding of the protocol. This is needed to make sure the translation is accurate both in its form and content. 

This is usually done within the Discord in each language specific channel. 

## Submitting the translation
Once the translation or update is done and has been reviewed, translators can head over to Angle's docs repository issues tab (https://github.com/AngleProtocol/angle-docs/issues) and submit a new issue. 

### Content of the issue
The issue should have a clear title indicating what is being translated, in which language, etc. 
The issue should also contain the discord handles of both the translator and reviewer, and a link to the original documentation page.  

Then, the main content should be the translated text written directly in the issue, in Markdown format. If you're not familiar with Markdown yet, you can learn more on this guide: https://www.markdownguide.org/cheat-sheet/. 

## Thank you for your contribution

Thanks to everyone helping to make Angle accessible to a wider audience!
